<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Kapil" failures="1" tests="4" name="Test" time="13.736" errors="0" timestamp="2024-06-25T14:19:18 EDT">
  <testcase classname="BlazeDemo.PracticeTable" name="test_01" time="0.114"/>
  <testcase classname="BlazeDemo.PracticeTable" name="test_02" time="3.672"/>
  <testcase classname="BlazeDemo.PracticeTable" name="test_03" time="0.003">
    <skipped/>
  </testcase> <!-- test_03 -->
  <testcase classname="BlazeDemo.PracticeTable" name="test_04" time="0.097">
    <failure type="java.lang.AssertionError" message="expected [2015.0] but found [2105.82]">
      <![CDATA[java.lang.AssertionError: expected [2015.0] but found [2105.82]
at BlazeDemo.PracticeTable.test_04(PracticeTable.java:152)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test_04 -->
</testsuite> <!-- Test -->
